ee/tasks/replay.py

```diff
@@ -70,3 +72,25 @@ def generate_recordings_embeddings_batch() -> None:
         except Exception as e:
             logger.error(f"[generate_recordings_embeddings_batch] Error: {e}.", exc_info=True, error=e)
             pass
+
+
+@shared_task(ignore_result=True, queue=CeleryQueue.SESSION_REPLAY_EMBEDDINGS.value)
+def generate_replay_embedding_error_clusters() -> None:
+    for team_id in settings.REPLAY_EMBEDDINGS_ALLOWED_TEAMS:
```

Comment:
while this list is small this is fine, but really we should queue a task for each team id so that celery gets a queue of tasks to run through rather than a single long task

not blocking here... maybe a TODO comment so we don't forget!

Reply:
Thought it worth updating while we were here. Should look a little better now

Reply:
20 good citizen points